### PR TITLE
Added contents of CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by suricactus**
This is a contributing,md file I made after compiling similar files in several OSGeo projects. Big influence from the QGIS `CONTRIBUTING.md`.

@mbernasocchi @marioba @m-kuhn should be good enough.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 6af0c3c9d2636bde10b26399b43d4983f01b14b2: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3274-5ce0b7cf87d755c3f2b682d15e30f213d18b5c8a-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3274-5ce0b7cf87d755c3f2b682d15e30f213d18b5c8a-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3274-5ce0b7cf87d755c3f2b682d15e30f213d18b5c8a-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3274-5ce0b7cf87d755c3f2b682d15e30f213d18b5c8a-x86-android.apk)
</details>

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
 # Contributing to QField
-QField is an open source project and we appreciate contributions very much. You can find all information on how to contribute to the project on our website https://docs.qfield.org/get-started/contribute/#development
+
+QField is an open source project and we appreciate contributions very much. Even if you are **not** a developer, there are multiple ways you can contribute to the project. Scroll to bottom if you are a developer.
```
Comment (marioba) [inline CONTRIBUTING.md:3]:
```suggestion
QField is an open source project and we appreciate contributions very much. Even if you are **not** a developer, there are multiple ways you can contribute to the project. Go directly to the section [New features, enhancements and bug fixes](#new-features-enhancements-and-bug-fixes) if you are a developer.
```

**Review by marioba — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,2 +1,33 @@
 # Contributing to QField
-QField is an open source project and we appreciate contributions very much. You can find all information on how to contribute to the project on our website https://docs.qfield.org/get-started/contribute/#development
+
+QField is an open source project and we appreciate contributions very much. Even if you are **not** a developer, there are multiple ways you can contribute to the project. Go directly to the section [New features, enhancements and bug fixes](#new-features-enhancements-and-bug-fixes) if you are a developer.
```
Comment (m-kuhn) [inline CONTRIBUTING.md:3]:
```suggestion
QField is an open-source project and we appreciate contributions very much. If you are a developer or not, there are multiple ways you can contribute to the project. Go directly to the section [New features, enhancements and bug fixes](#new-features-enhancements-and-bug-fixes) if you are a developer.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,2 +1,33 @@
...
+You can help us by submitting bug reports or verification of bugs in the [QField issue tracker](https://github.com/opengisch/QField/issues).
+
+## Documentation
+
+Join the documentation effort about QField. Add, improve or translate the QField documentation in your language.
+Check [QField-docs repository](https://github.com/opengisch/QField-docs#qfield-documentation) for more information.
+
+## Translations
+
+Please help translate QField to your language. Community efforst already made QField available in multiple languages. However, translations are an ongoing effort with the fast pace development of QField, so regularly check your language. Learn more [how to become a translator on Transifex](https://docs.transifex.com/getting-started-1/translators#joining-a-translation-team).
```
Comment (m-kuhn) [inline CONTRIBUTING.md:16]:
```suggestion
Please help translate QField to your language. Community efforts already made QField available in multiple languages. However, translations are an ongoing effort with the fast pace development of QField, so regularly check your language. Learn more [how to become a translator on Transifex](https://docs.transifex.com/getting-started-1/translators#joining-a-translation-team).
```

**Pull request merged**